samples/DataSourceWithProcessingAndExtensions/DataGenerator/Jsonbinary.cs

```diff
@@ -0,0 +1,41 @@
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace DataGenerator
+{
+    public static class JsonBinary
+    {
+        public static byte[] ToBinary(object value)
+        {
+            byte[] result = null;
+
+            using (var stream = new MemoryStream())
+            {
+                using (var writer = new BinaryWriter(stream, Encoding.Default, true))
```

Comment:
For .NET Core, this is always UTF8.
For other versions of the framework, there are no guarantees about what it is, and it can change on a single computer.
Given that this is sample code, I think we should err on the side of caution and just use the explicit UTF8 encoding property.

https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.default?view=netcore-3.1